website/docs/troubleshooting/postgres/upgrade_kubernetes.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Upgrading PostgreSQL on Kubernetes
+---
+
+## Placeholders
+
+-   `authentik-postgresql-0` is the Kubernetes Pod running PostgreSQL
+
+### Prerequisites
+
+This migration requires some downtime, in which authentik has to be stopped. To do this, run
+
+```shell
+kubectl scale deploy --replicas 0 authentik-server
+kubectl scale deploy --replicas 0 authentik-worker
+```
+
+### Dumping the current database
+
+Run `kubectl exec -it authentik-postgresql-0 -- bash` to get a shell in the PostgreSQL pod.
+
+Run the following commands to dump the current data into a `.sql` file:
+
+```shell
+# This is the path where the PVC is mounted, so we'll place the dump here too
+cd /bitnami/postgresql/
+# Set the postgres password based on the `POSTGRES_POSTGRES_PASSWORD` environment variable
+export PGPASSWORD=$POSTGRES_POSTGRES_PASSWORD
+# Dump the authentik database into an sql file
+pg_dump -U postgres $POSTGRES_DB > dump-11.sql
+```
+
+### Stopping PostgreSQL and starting the upgrade
+
+To upgrade, change the following entries in your `values.yaml` used to deploy authentik:
+
+```yaml
+postgresql:
+    diagnosticMode:
+        enabled: true
+    image:
+        tag: 15.2.0-debian-11-r26
+```
+
+And run `helm upgrade --install authentik authentik/authentik -f values.yaml` to apply these changes. Depending on your configuration, you might have to repeat the steps from [Prerequisites](#prerequisites).
+
+After the upgrade is finished, you should have a new PostgreSQL pod running with the updated image.
+
+### Removing the old data
+
+As the PVC mounted by the PostgreSQL pod still contains the old data, we need to remove/rename that data, so that PostgreSQL can initialize it with the new version.
```

Comment:
```suggestion
Because the PVC mounted by the PostgreSQL pod still contains the old data, we need to remove/rename that data, so that PostgreSQL can initialize it with the new version.
```